egs/csj/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
+
+. cmd.sh
+
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+if [ -e data/train_dev ] ;then
+    dev_set=train_dev
+fi
+
+stage=0
+train_stage=-10 
+dir=exp/nnet3/nnet_tdnn_a
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1 
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA 
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+local/nnet3/run_ivector_common.sh --stage $stage || exit 1;
+
+if [ $stage -le 8 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+    utils/create_split_dir.pl \
+     /export/b0{3,4,5,6}/$USER/kaldi-data/egs/csj-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
+  fi
+
+  time steps/nnet3/train_tdnn.sh --stage $train_stage \
+    --num-epochs 8 --num-jobs-initial 2 --num-jobs-final 10 \
+    --splice-indexes "-4,-3,-2,-1,0,1,2,3,4  0  -2,2  0  -4,4 0" \
+    --feat-type raw \
+    --online-ivector-dir exp/nnet3/ivectors_train_nodup \
+    --cmvn-opts "--norm-means=false --norm-vars=false" \
+    --io-opts "-tc 12" \
+    --initial-effective-lrate 0.005 --final-effective-lrate 0.0005 \
+    --cmd "$decode_cmd" \
+    --pnorm-input-dim 2000 \
```

Comment:
given the amount of data you could probably increase pnorm-input-dim and pnorm-output-dim.  I would recommend the `--splice-indexes` used in swbd recipe and using `--relu-dim` which has been showing better results in recent experiments. Of course these are only suggested changes and it might be better to keep these settings  as they reproduce the results.